odilia/src/main.rs

```diff
@@ -90,32 +93,23 @@ async fn sigterm_signal_watcher(
 
 use atspi::events::document::LoadCompleteEvent;
 
-#[tracing::instrument]
-async fn change_priority(
-	SpeechPriority(priority): SpeechPriority,
-	Speech(ssip): Speech,
-) -> Result<(), odilia_common::errors::OdiliaError> {
-	ssip.send(SSIPRequest::SetPriority(priority)).await?;
-	Ok(())
-}
-
 #[tracing::instrument]
 async fn speak(
```

Comment:
I will make a service/layer to handle the commands. But I want to handle them like this as I believe it will be more flexible  especially with addons later on.